src/solvers/smt2/smt2_parser.h

```diff
@@ -73,13 +74,35 @@ class smt2_parsert:public smt2_tokenizert
   irep_idt get_fresh_id(const irep_idt &);
   irep_idt rename_id(const irep_idt &) const;
 
+  struct signature_with_parameterst
```

Comment:
with_parameter_ids? The actual type already has parameters

Reply:
ok